.changeset/big-oranges-behave.md

```diff
@@ -0,0 +1,8 @@
+---
+'@example/blog': minor
+'@example/docs': minor
+'@example/with-markdown': minor
+'@astrojs/markdown-remark': minor
```

Comment:
I would also bump `astro` with a `minor` as well since that's where this change has more surface area.